lib/beaker/options/command_line_parser.rb

```diff
@@ -88,6 +88,12 @@ def initialize
             @cmd_options[:preserve_hosts] = mode || 'always'
           end
 
+          opts.on '--debug-errors',
```

Comment:
While I only want this option to be on the _run_ subcommand; beaker will raise an exception if I pass a command through to beaker that's not declared here.
I could set the hide attribute on this option so it doesn't appear in beaker --help